mage_ai/data_preparation/models/block/integration/__init__.py

```diff
@@ -246,6 +254,14 @@ def __update_col_names(schema_dict: Dict, properties_key: str, new_columns: List
                                         if k in properties_original else v
                                         for k, v in properties_updated.items()
                                     }
+                                    # Separately, update schema types from alias map
+                                    if len(output_col_aliases.keys()):
+                                        for cname in schema_updated['schema']['properties']:
```

Comment:
I think it should be `for cname in schema_updated['schema']['properties'].keys()`

Reply:
Sure, updated